data/processed/port_wsc.html

```diff
@@ -3941,38 +3942,38 @@
         </li>
         <p></p>
         <li>
-            At the Loebner competition the judges couldn't figure out which respondents
-            were the chatbots because
-            <b> they </b> were so stupid.
-            <p></p><em> Trecho: </em><b> they </b> were so stupid.
+            Na competição Loebner os juízes não conseguiam descobrir que respondentes
+            eram os <i> chatbots </i> porque
+            <b> eles </b> eram muito estúpidos.
+            <p></p><em> Trecho: </em><b> eles </b> eram muito estúpidos.
             <p></p>
             <ol type="A">
-                <li> the judges </li>
-                <li> the chatbots </li>
+                <li> Os juízes </li>
+                <li> Os <i>chatbots</i> </li>
             </ol>
             <p><b> Resposta Correta: </b> A. <br><b> Fonte: </b> Sabri Meeran
             </p>
         </li>
         <p></p>
         <li>
-            At the Loebner competition the judges couldn't figure out which respondents
-            were the chatbots because
-            <b> they </b> were so advanced.
-            <p></p><em> Trecho: </em><b> they </b> were so advanced.
+            Na competição Loebner os juízes não conseguiam descobrir que respondentes
+            eram os <i> chatbots </i> porque
+            <b> eles </b> eram muito avançados.
+            <p></p><em> Trecho: </em><b> eles </b> eram muito avançados.
             <p></p>
             <ol type="A">
-                <li> the judges </li>
-                <li> the chatbots </li>
+                <li> Os juízes </li>
+                <li> Os <i>chatbots</i> </li>
             </ol>
             <p><b> Resposta Correta: </b> B. <br><b> Fonte: </b> Sabri Meeran
             </p>
         </li>
         <p></p>
         <li>
-            The user changed his password from "GrWQWu8JyC" toÂ&nbsp;"willow-towered Canopy
-            Huntertropic wrestles" as
-            <b> it </b> was easy to forget.
-            <p></p><em> Trecho: </em><b> it </b> was easy to forget.
+            O usuário mudou sua senha "GrWQWu8JyC" para "willow-towered Canopy
```

Comment:
será que não vale a pena traduzir para o português também a senha? (até pq em inglês, há o risco de dar "out of vocabulary" em relaćão ao vocabulário que nosso modelo conhece)

Reply:
O duro é que essa frase é muito bizarra. Não como traduzi-la. :/